docs/resources/gateway_config_store_secret.md

```diff
@@ -0,0 +1,45 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "konnect_gateway_config_store_secret Resource - terraform-provider-konnect"
+subcategory: ""
+description: |-
+  GatewayConfigStoreSecret Resource
+---
+
+# konnect_gateway_config_store_secret (Resource)
+
+GatewayConfigStoreSecret Resource
+
+## Example Usage
+
+```terraform
+resource "konnect_gateway_config_store_secret" "my_gatewayconfigstoresecret" {
+  config_store_id  = "d32d905a-ed33-46a3-a093-d8f536af9a8a"
+  control_plane_id = "9524ec7d-36d9-465d-a8c5-83a3c9390458"
+  key              = "ConfigStoreSecretKey"
```

Comment:
Looks like updating the key leads to creation of new config store secret, and returns a 201. We should use `x-speakeasy-param-force-new` for `key` perhaps?

Reply:
Good catch. Can you update the source file in platform-api and regenerate?

Reply:
Yes, will do.